### PR TITLE
Remove old links to images in sample data testing plugin

### PR DESCRIPTION
**Pull request by ghazal**
### Summary of Changes

In /plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.ini, removal of (obsolete) links to (ugly)  images in (obsolete admin template) hathor.

### CODE REVIEW

**Comment by brianteeman** [comment]
The sampledata plugin is being rewritten in the joomla/cassiopeia repository so I wouldnt do anything with this for now

**Comment by richard67** [comment]
> The sampledata plugin is being rewritten in the joomla/cassiopeia repository so I wouldnt do anything with this for now

@brianteeman As far as I know we handle only **_blog_** sample data there, but this issue here is about **_testing_** sample data.

@chmst Is my understanding right, or do you plan to handle testing sample data there, too?

**Comment by ghazal** [comment]
It's a simple error, not a super important piece of code. Why leave it as it is while the plugin is being rewritten?

BTW, there are outdated links to beez images as well.

**Comment by chmst** [comment]
We have 2 plugins: 

Blog sample data are rewrittenin cassiopeia.

Testing sample data must be reviewed, in J4, as new features also come with new data. So it does not make much sense to start with a single link. (But this one is really bad :) )

**Comment by brianteeman** [comment]
Sorry I didnt see this was for the testing data. There is already an issue open for tracking that. Its never shipped with Joomla and its majorly broken as it is right now.

**Comment by richard67** [comment]
@brianteeman No problem. And thanks for the hint, I found the issue: #25777 .

Closing this one here as duplicate because it's covered by #25777 .

@ghazal thanks for reporting.

**Comment by richard67** [comment]
Silly me, it's a PR and not an issue => Re-opening.

**Comment by chmst** [comment]
Yes it is broken. I hade a chat with @hackwar and  will work on it, but it is a huge chanllenge.

**Comment by richard67** [comment]
It this PR fixes only a part of the 3rd item in to-do list of issue #25777 , is that right? Or does it fix it all?

**Comment by gostn** [comment]
need label:"Information Required"

**Comment by brianteeman** [comment]
Can someone update the title please to show its for the testing plugin

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 789901ddb3db32a65364bbd4a9428f3a20e0744a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31045">issues.joomla.org/tracker/joomla-cms/31045</a>.</sub>

**Comment by chmst** [comment]
When this is merged we can remove old sample data images too.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31045">issues.joomla.org/tracker/joomla-cms/31045</a>.</sub>

**Comment by drmenzelit** [comment]
I have tested this item :white_check_mark: successfully on 789901ddb3db32a65364bbd4a9428f3a20e0744a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31045">issues.joomla.org/tracker/joomla-cms/31045</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31045">issues.joomla.org/tracker/joomla-cms/31045</a>.</sub>

File: plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.ini [hunk 1]
```diff
@@ -129,7 +129,7 @@ PLG_SAMPLEDATA_TESTING_SAMPLEDATA_CONTENT_ARTICLE_19_FULLTEXT=""
...
+PLG_SAMPLEDATA_TESTING_SAMPLEDATA_CONTENT_ARTICLE_20_INTROTEXT="<p><There are lots of places you can get help with Joomla!. In many places in your site administrator you will see the help icon. Click on this for more information about the options and functions of items on your screen. Other places to get help are:</p><ul><li><a href='http://forum.joomla.org'>Support Forums</a></li><li><a href='https://docs.joomla.org'>Documentation</a></li><li><a href='http://resources.joomla.org'>Professionals</a></li><li><a href='http://shop.joomla.org/amazoncom-bookstores.html'>Books</a></li></ul>"
```
Comment (Quy) [inline plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.ini:132]: Remove `<` from `<There`.

**Comment by richard67** [comment]
Back to pending.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31045">issues.joomla.org/tracker/joomla-cms/31045</a>.</sub>

File: plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.ini (hunk 1 above)
Comment (Quy) [reply plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.ini:132]: The branch is not accessible/available. Maybe merge it and do a followup PR if it is not possible to submit the change.

**Comment by richard67** [comment]
RTC again<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31045">issues.joomla.org/tracker/joomla-cms/31045</a>.</sub>

File: plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.ini (hunk 1 above)
Comment (richard67) [reply plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.ini:132]: Right. I didn't notice that. Do you wanna merge and make the follow up PR?

**Pull request merged**